### PR TITLE
is_null -> empty, !is_null-> isset. That's really much faster

### PR DESCRIPTION
**Pull request by GSokol**

### CODE REVIEW

File: src/ReCaptcha/RequestMethod/Socket.php [hunk 1]
```diff
@@ -47,7 +47,7 @@ class Socket
      */
     public function fsockopen($hostname, $port = -1, &$errno = 0, &$errstr = '', $timeout = null)
     {
-        $this->handle = fsockopen($hostname, $port, $errno, $errstr, (is_null($timeout) ? ini_get("default_socket_timeout") : $timeout));
+        $this->handle = fsockopen($hostname, $port, $errno, $errstr, (empty($timeout) ? ini_get("default_socket_timeout") : $timeout));
```
Comment (rowan-m) [inline src/ReCaptcha/RequestMethod/Socket.php]: I'd prefer `!isset()` here and on #L68 as I think that makes the intent more explicit.

**Comment by rowan-m** [comment]
@GSokol ping in case you can check the CLA and comments

**Comment by GSokol** [comment]
sorry for so late!

File: src/ReCaptcha/RequestMethod/Socket.php (hunk 1 above)
Comment (GSokol) [inline src/ReCaptcha/RequestMethod/Socket.php]: So the better way is `!== null`

**Comment by rowan-m** [comment]
Having come back to this about 100 years later, I'm actually happier keeping the `is_null` checks in there as they are more explicit and readable. I'm not overly fond of relying on PHP's behaviour in treating `null` as not set, so I prefer the clearer intention in directly checking for null.

**Pull request closed without merging**